packages/ui/src/components/va-skip-link/VaSkipLink.stories.ts

```diff
@@ -0,0 +1,60 @@
+import { VaSkipLink } from './'
+
+export default {
+  title: 'VaSkipLink',
+  component: VaSkipLink,
+}
+
+export const Target = () => ({
```

Comment:
Needs default demo. (this one seems pretty default to me) skip link doesn't work without target and slot.